### ISSUE TITLE
貌似有个小bug，输出的学习速率lr不是实时的

### ISSUE THREAD

**Issue by dcsaf** [tanluren/yolov3-channel-and-layer-pruning#25]
print('learning rate:',optimizer.param_groups[0]['lr']) 是在调整学习速率
lr = adjust_learning_rate(optimizer, 0.1, epoch, ni, nb)之前执行，打印的lr不是当前的lr，而是上一个epoch的学习速率lr

**Comment by tanluren** [comment]
看的很仔细，这是之前用scheduler时写的，后来用warmup忘记改了。已修正，谢谢哈
